poi/src/main/java/org/apache/poi/ss/usermodel/Workbook.java

```diff
@@ -220,6 +222,32 @@ Licensed to the Apache Software Foundation (ASF) under one or more
      */
     Iterator<Sheet> sheetIterator();
 
+    /**
+     * Alias for {@link #sheetIterator()} to allow foreach loops
+     */
+    @Override
+    default Iterator<Sheet> iterator() {
+        return sheetIterator();
+    }
+
+    /**
+     *  Returns a spliterator of the sheets in the workbook
+     *  in sheet order. Includes hidden and very hidden sheets.
+     *
+     * @return a spliterator of the sheets.
+     */
+    default Spliterator<Sheet> sheetSpliterator() {
```

Comment:
maybe, we only need spliterator() - I know we have 2 iterator methods but that is some legacy stuff - I don't think new stuff needs to add 2 separate methods

Reply:
ok, will remove it, I agree its better to have 1 method

Reply:
Done